metrics/expvar/expvar.go

```diff
@@ -75,9 +81,35 @@ type histogram struct {
 	hist *hdrhistogram.WindowedHistogram
 
 	name   string
+	count  metrics.Counter
 	gauges map[int]metrics.Gauge
 }
 
+// Name returns the name of the histogram
+func (h *histogram) Name() string {
+	return h.name
+}
```

Comment:
Yes, I'm pretty sure I agree with you, just need to think through the consequences :)